play-app/src/Components/Cartcard/Cartcard.jsx

```diff
@@ -0,0 +1,78 @@
+import "./Cartcard.css"
+import {GoPlus} from "react-icons/go";
+import {BiMinus} from "react-icons/bi";
+import { useCart } from "../../Context/CartCont";
+import axios from "axios";
+import { itemWishlist } from "../../Utils/CartUtils/itemWishlist";
```

Comment:
In useCart and itemWishlist import you can index.js for each folder

Reply:
will do that